src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -5,12 +5,20 @@ public class AuthenticationService {
      * Imagine that some user wants to login to your site.
      * You should check if user credentials (login and password) are valid or not.
      * All users are stored in <code>UserService</code> class.
-     * @param email - user's email
+     *
+     * @param email    - user's email
      * @param password - user's password
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+        UserService user = new UserService();
+        if (user.findByEmail(email) != null) {
+            if (user.findByEmail(email).getEmail().equals(email)
+                    && user.findByEmail(email).getPassword().equals(password)) {
+                return true;
+            }
+        }
         return false;
```

Comment:
1. don't create a service object per each method call - make it a class field
2. Don't call UserService object user - it's very confusing since we have model User
3. Save the result of userService.findByEmail to a local variable - your method will be 3 times more efficient :)
4. Try to achieve the same without using boolean literals like true and false :)